src/main/webapp/resources/js/apis/analysis/analysis.js

```diff
@@ -3,11 +3,9 @@
  */
 import axios from "axios";
 
-// Ajax URL for Analysis
-const ANALYSIS_URL = `${window.TL.BASE_URL}ajax/analysis`;
+const ANALYSES_URL = `ajax/analyses`;
 
-// Ajax URL for Analyses
-const ANALYSES_URL = `${window.TL.BASE_URL}ajax/analyses`;
+const ANALYSIS_URL = `/ajax/analysis`;
```

Comment:
Don't need the leading `/`, it gets all sorted by the ajax interceptor.  You don't need to remove it here either because it handles that as well.